components/net/http_loader.rs

```diff
@@ -1103,7 +1079,7 @@ fn http_network_fetch(request: &Request,
     // do not. Once we support other kinds of fetches we'll need to be more fine grained here
     // since things like image fetches are classified differently by devtools
     let is_xhr = request.destination == Destination::None;
-    let wrapped_response = obtain_response(&factory, &url, &request.method,
+    let wrapped_response = obtain_response(context.connector.clone(), &url, &request.method,
```

Comment:
AFAIK this clone is pointless, and you can pass `&Pool<Connector>` to `obtain_response`.